repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
```

Comment:
what about service-resource-roles (for determining where the kubernetes
dcos service runs) vs. task-resource-roles (how k8s-mesos launches tasks)?

I don't really like that a lot either. Is the goal just to run apiserver on
a public slave? Or do we really want the etcd, scheduler, and
controller-manager running there all the time too? Once we get the
containers split up, I'd rather just the "kubectl proxy" (apiserver proxy)
ran on the public slave and the rest on a private slave.

What about "api-resource-roles" (just drop 'server')?


On Wed, Sep 30, 2015 at 5:55 PM, Karl Isenberg <notifications@github.com>
wrote:

> In repo/packages/K/kubernetes/1/config.json
> <https://github.com/mesosphere/multiverse/pull/13#discussion_r40857607>:
>
> > @@ -81,6 +81,15 @@
> >            "minimum": 0,
> >            "type": "integer"
> >          },
> > +        "resource-roles": {
>
> I was thinking apiserver-resource-roles, but I'm not sure that's much
> clearer, given that the other apiserver configs are flags sent to apiserver.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/multiverse/pull/13/files#r40857607>.
>